src/libs/ajax.js

```diff
@@ -28,9 +29,9 @@ window.ajaxOverrideUtils = {
   makeSuccess: body => _wrappedFetch => () => Promise.resolve(new Response(JSON.stringify(body), { status: 200 }))
 }
 
-const authOpts = (token = getUser().token) => ({ headers: { Authorization: `Bearer ${token}` } })
-const jsonBody = body => ({ body: JSON.stringify(body), headers: { 'Content-Type': 'application/json' } })
-const appIdentifier = { headers: { 'X-App-ID': 'Saturn' } }
+export const authOpts = (token = getUser().token) => ({ headers: { Authorization: `Bearer ${token}` } })
+export const jsonBody = body => ({ body: JSON.stringify(body), headers: { 'Content-Type': 'application/json' } })
+export const appIdentifier = { headers: { 'X-App-ID': 'Saturn' } }
```

Comment:
Maybe all this common stuff could go in another module (like ajax/ajax-common.js or something like that) to avoid circular imports.